src/user.ts

```diff
@@ -3,7 +3,7 @@ import { IDContract } from './services/contracts/IDContract';
 import { Bytes, KeyType, Name, PrivateKey } from '@greymass/eosio';
 import { randomBytes, randomString, sha256 } from './util/crypto';
 import { createSigner } from './services/eosio/transaction';
-import argon2 from 'argon2';
+import scrypt from "scrypt-js";
```

Comment:
https://medium.com/analytics-vidhya/password-hashing-pbkdf2-scrypt-bcrypt-and-argon2-e25aaf41598e
https://crypto.stackexchange.com/a/51523

Check out these, I'd prefer to stack with argon2 if possible and even better to upgrade to argon2id as suggested

Can we try with this, an isomorphic (cross-environment) version, with argon2id?
https://www.npmjs.com/package/@sphereon/isomorphic-argon2

Reply:
i have tested the package and just like all the other it uses pollyfill and config expo config https://github.com/react-native-community/cli/issues/870

unfortunately wasm isn't available for RN and there isn't any package that can implement that in pure javascript without crypto package